dbt_subprojects/hourly_spellbook/models/_sector/blockchain_metrics/_schema.yml

```diff
@@ -13,3 +13,19 @@ models:
           combination_of_columns:
             - blockchain
             - block_hour
+
+  - name: evms_transaction_address_metrics
+    meta:
+      blockchain: evms
+      sector: metrics
+      contributors: jeff-dude
+    config:
+      tags: [ 'evms','metrics', 'address' ]
+    description: "fundamental blockchain transaction address metrics"
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - blockchain
+            - block_hour
+            - from_address
+            - to_address
```

Comment:
Probably more of a comment on the requirements than on the PR itself but I would have expected these address metrics to have just 1 address as the subject of the data instead of a (from, to) address pair as it currently is defined.
Currently the `tx_count` column is '1' for ~90% of the rows meaning we mainly just have duplicate data from evms.transactions in here. 
![image](https://github.com/user-attachments/assets/d8ab328c-8a54-465f-a038-808035b5609c)
https://dune.com/queries/4020812/6768658

Reply:
i have confirmed with hannah the from/to both existing is the intended output. there are use cases it enables for their usage downstream.